readme.md

```diff
@@ -137,6 +137,8 @@ MERGED #60 Feature 3
 
 To merge only part of the stack use the **--upto** flag with the top pull request number in the stack that you would like to merge.
 
+If you want to merge changes using a method other than the default "rebase", this can be configured in the [Repository Config](#Configuration) as `mergeMethod`.
```

Comment:
How about the following: 
"By default merges are done using the rebase merge method, this can be changed using the `mergeMethod` configuration."

(since the config section is right below, I don't think we need to link to it here)